src/alert/interfaces.ts

```diff
@@ -40,6 +71,9 @@ export interface AlertProps extends BaseComponentProps {
   /**
    * Adds an aria-label to the dismiss button.
    * @i18n
+   *
+   * @deprecated Use the label properties inside `i18nStrings` instead.
```

Comment:
here it's just one property, so can be "Use `i18nStrings.dismissAriaLabel` instead."